lms/djangoapps/teams/tests/test_views.py

```diff
@@ -2657,3 +2657,78 @@ def test_upload_learners_exceed_max_team_size(self):
         )
         response_text = json.loads(response.content.decode('utf-8'))
         self.assertEqual(response_text['errors'][0], 'Team {} is full.'.format(team1))
+
+    def test_deletion_via_upload_csv(self):
+        self.test_upload_valid_csv_simple()
+        username = 'a_user'
+        topic_0_id = 'topic_0'
+        self.assertEqual(len(CourseTeamMembership.objects.filter(
+            user_id=self.users[username].id,
+            team__topic_id=topic_0_id
+        )), 1)
+        csv_content = 'user,mode,{},topic_1'.format(topic_0_id) + '\n'
+        csv_content += '{},audit'.format(username)
+        csv_file = SimpleUploadedFile('test_file.csv', csv_content.encode('utf8'), content_type='text/csv')
+        self.client.login(username=self.users['course_staff'].username, password=self.users['course_staff'].password)
+        self.make_call(
+            reverse('team_membership_bulk_management', args=[self.good_course_id]),
+            201,
+            method='post',
+            data={'csv': csv_file},
+            user='staff'
+        )
+        self.assertEqual(len(CourseTeamMembership.objects.filter(
+            user_id=self.users[username].id,
+            team__topic_id=topic_0_id
+        )), 0)
+
+    def test_reassignment_via_upload_csv(self):
+        self.test_upload_valid_csv_simple()
+        username = 'a_user'
+        topic_0_id = 'topic_0'
+        nuclear_team_name = 'team nuclear power'
+        self.assertEqual(len(CourseTeamMembership.objects.filter(
+            user_id=self.users[username].id,
+            team__topic_id=topic_0_id
+        )), 1)
+        csv_content = 'user,mode,{}'.format(topic_0_id) + '\n'
+        csv_content += '{0},audit,{1}'.format(username, nuclear_team_name)
+        csv_file = SimpleUploadedFile('test_file.csv', csv_content.encode('utf8'), content_type='text/csv')
+        self.client.login(username=self.users['course_staff'].username, password=self.users['course_staff'].password)
+        self.make_call(
+            reverse('team_membership_bulk_management', args=[self.good_course_id]),
+            201,
+            method='post',
+            data={'csv': csv_file},
+            user='staff'
+        )
+        self.assertEqual(len(CourseTeamMembership.objects.filter(
+            user_id=self.users[username].id,
+            team__name=nuclear_team_name
```

Comment:
since you're filtering on specific team name here, maybe do the same above instead of by topic (or in addition to), and then also assert below that the old membership no longer exists?